README.md

```diff
@@ -1,13 +1,28 @@
+The blue requirements were:
+- Your API should implement the routes exactly as documented in the brief.
+- Your GET /thoughts endpoint should only return 20 results, ordered by createdAt in descending order.
+- Your API should validate user input and return appropriate errors if the input is invalid.
+- In the POST /thoughts endpoint to create a new thought, if the input was invalid and the API is returning errors, it should set the response status to 400 (bad request).
+- The endpoint to add hearts to a thought should return an appropriate error if the thought was not found.
 
 ## The problem
+1. I tackled the problem by reffering to the previous weeks project to get the database up and running and to start to create the model which is the blueprint for the object that is created everytime the user posts a new message.
+2. Because each property of the model required some extra parameters and requirements. I got those done before I went on to create the endpoints. These extra parameters were to both help with validation when the user is posting a new thought and to set the default for both the hearts and createdAt to a specific value so they couldn't be overwritten by the user when they were creating a new thought.
+3. Then I focused on getting the endpoints written and implementing what was required for each endpoint according to the project. This included:
+    - Using the sort and limit methods to make sure that the last 20 thoughts were returned on the get endpoint.
+    - That only the message property can be updated when the post request is sent to the backend. 
+    - And that the thought id is targeted using the mongoose method of updateOne() and that the hearts property is incremented by 1 by using the $inc  and +1. 
+4. I used the try and catch form to allow for the handling of the requests if they succeed or not. And implemented in the catch an error to be shown if for example the user tries to send a message that is less than 5 and more than 140 letters long or heart a thought with an invalid id.
+5. Once everything was set up I tested it in postman using the local host, made sure that the data was coming through on my database and then depolyed to heroku. 
+6. Then I changed the endpoints in the Happy thoughts project and made sure that they were working and that I was getting the data that was stored on and being posted to my database.
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
-
-## View it live
+## View it live 💌
+Happy thoughts API: https://claireshappythoughts.herokuapp.com/
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+Happy thoughts project: https://silly-wing-fdef06.netlify.app/
```

Comment:
Incredibly great job Claire! So understandable and clear! I love the way how you write you the read.me files! Congrats!